File: Projekat/src/app/entities/rate/rate.ts
import { DatePipe } from '@angular/common';

export class Rate {
    endDay: DatePipe;
    startDay: DatePipe;
    id: string;
    vehicle: string;
    serviceName: string;
    serviceRating: string;
    vehicleRating: string;

    constructor(endDay: DatePipe, startDay: DatePipe,  id:string, vehicle: string, serviceName: string, serviceRating: string, vehicleRating: string){
        this.endDay = endDay;
        this.startDay = startDay;
        this.id = id;
        this.serviceName = serviceName;
        this.vehicle = vehicle;
        this.serviceRating = serviceRating;
        this.vehicleRating = vehicleRating;
    }
}

File: ProjekatApi/ProjekatApi/FormModel/Rate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjekatApi.FormModel
{
    public class Rate
    {
        public string id { get; set;}
        public DateTime startDay { get; set; }
        public DateTime endDay { get; set; }
        public string serviceName { get; set; }
        public string vehicle { get; set; }
        
    }
}
